app/representers/member_permissions_representer.rb

```diff
@@ -57,8 +57,8 @@ class XML < MemberPermissionsRepresenter
     include Roar::XML
     wraps_resource :permissions
 
-    collection :allowed_sections, as: :allowed_section, wrap: :allowed_sections, getter: ->(opts) { opts[:user].member_permission_ids }
-    collection :allowed_service_ids, as: :allowed_service_id, wrap: :allowed_service_ids, getter: ->(opts) { opts[:user].member_permission_service_ids }
+    collection :allowed_sections, as: :allowed_section, wrap: :allowed_sections, getter: ->(opts) { opts[:user].allowed_sections&.sort }
+    collection :allowed_service_ids, as: :allowed_service_id, wrap: :allowed_service_ids, getter: ->(opts) { opts[:user].allowed_service_ids }
```

Comment:
This is not a functional change, just for the consistency sake now XML and JSON representer use the same methods:
`allowed_sections` which is alias for `member_permission_ids`
and
`allowed_service_ids` which is alias for `member_permission_service_ids`

Reply:
The functional changes is adding `&.sort`

Reply:
I don't like fixing tests by changing functionality. So better make test order agnostic.

Or if we want sort order, then better specify it in the active record query/scope instead of doing within ruby. Would be better also for pagination if ever implemented.

Reply:
You're too fast :grimacing: I'm writing a description...